Makefile

```diff
@@ -28,7 +28,7 @@ test_lint:
 	}
 
 test_go:
-	go test -race -p=1 ./...
+	go test -race -p=1 -short ./...
```

Comment:
Some of the skipped tests look super simple. But maybe I missed their complexity on first sight.